tests/scripts/import_book.py

```diff
@@ -125,11 +133,7 @@ class Meta:
     do_import(book_resource, dataset)
 
 
-def main():
-    do_create()
-    #do_update()
+def run():
+    #do_create()
+    do_update()
     #do_delete()
-
-
-if __name__ == "__main__":
-    main()
```

Comment:
if removing this and recommending `./manage.py runscript import_book` from `django-extensions`, the `django.setup()` line is no longer required, right?

Reply:
good spot, I have removed this extraneous line.